tests/utils/test_utils.py

```
@@ -921,3 +923,232 @@ def fake_load(path):
+    reason="release either predates the extraction filters or predates CPython gh-107845, "
+           "so tar_extract_kwargs answers with something other than the filter's name")
+
+
+@needs_working_filters
+def test_tar_extract_kwargs():
+    assert tar_extract_kwargs() == {"filter": "data"}
+
+
+@needs_filters
+@pytest.mark.parametrize("filter", ("tar", "fully_trusted"))
+def test_tar_extract_kwargs_other_filters(filter):
```

Ruff (0.16.1)

[error] Function argument `filter` is shadowing a Python builtin

(A002)